### ISSUE TITLE
lr-scheduler for training the DynamicConv

### ISSUE THREAD

**Issue by getao** [pytorch/fairseq#768]
I noticed there are two values (cosine, and inverse_sqrt) for lr-scheduler in the examples given in https://github.com/pytorch/fairseq/blob/master/examples/pay_less_attention_paper/README.md. I wonder which one is correct?

# Training
SAVE="save/dynamic_conv_wmt16en2de"
mkdir -p $SAVE
python -m torch.distributed.launch --nproc_per_node 8 $(which fairseq-train) \
    data-bin/wmt16_en_de_bpe32k --fp16  --log-interval 100 --no-progress-bar \
    --max-update 30000 --share-all-embeddings --optimizer adam \
    --adam-betas '(0.9, 0.98)' **--lr-scheduler inverse_sqrt** \
    --clip-norm 0.0 --weight-decay 0.0 \
    --criterion label_smoothed_cross_entropy --label-smoothing 0.1 \
    --min-lr 1e-09 --update-freq 16 --attention-dropout 0.1 --keep-last-epochs 10 \
    --ddp-backend=no_c10d --max-tokens 3584 \
    **--lr-scheduler cosine** --warmup-init-lr 1e-7 --warmup-updates 10000 \
    --lr-shrink 1 --max-lr 0.001 --lr 1e-7 --min-lr 1e-9 --warmup-init-lr 1e-07 \
    --t-mult 1 --lr-period-updates 20000 \
    --arch lightconv_wmt_en_de_big --save-dir $SAVE \
    --dropout 0.3 --attention-dropout 0.1 --weight-dropout 0.1 \
    --encoder-glu 1 --decoder-glu 1

**Comment by lematt1991** [comment]
`cosine` is the one that will get used.  Later arguments override earlier appearing arguments for argparse.  We will update the README to avoid confusion.
